CMakeLists.txt

```diff
@@ -212,6 +216,9 @@ set(ENABLE_OSCAP_UTIL_VM TRUE CACHE BOOL "enables oscap util VM utility")
 # ENABLES OSCAP UTIL CHROOT
 set(ENABLE_OSCAP_UTIL_CHROOT TRUE CACHE BOOL "enables oscap util chroot")
 
+# ENABLES TESTS
+set(ENABLE_TESTS ${IS_LINUX} CACHE BOOL "enables tests")
```

Comment:
why are tests only for Linux? this seems like the wrong workaround for the issues. or is it because the tests are powered by bash? either way this should be described here